google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BigtableTracer.java

```diff
@@ -93,4 +95,12 @@ public void grpcChannelQueuedLatencies(long queuedTimeMs) {
   public void grpcMessageSent() {
     // noop
   }
+
+  /**
+   * Record the deadline when the request is sent to the Bigtable server. This will be called in
```

Comment:
the method name and comment need to be updated. "Record the operation timeout from user settings for calculating remaining deadline." and change the method name to maybe `recordOperationTimeout` ?